src/util/sandbox/fd_sandbox.c

```diff
@@ -26,7 +26,7 @@
 #error "Target operating system is unsupported by seccomp."
 #endif
 
-#if !defined(__x86_64__)
+#if !defined(__x86_64__) && !defined(__aarch64__)
```

Comment:
Is this the only change needed? Doesn't the compiler need to produce a bit of different instructions in the programs as well?

Reply:
I know there are obscure bugs in a lot of seccomp programs around like, x86 syscall numbers having 2 variants and things like that.

Reply:
@mmcgee-jump This is the only change needed to unbreak the arm64 build with default flags. There are certainly other problems with Arm support (assumptions on a TSO memory model in our codebase), and the seccomp policies are all untested. But we've had reports that 'fddev' ran fine with this patch at one point.